README.md

```diff
@@ -1,5 +1,44 @@
 # IT Jobb nära dig
+Vårt mål är att skapa en webbapplikation som gör det möjligt att hitta IT-jobb beroende av ens geografiska plats.
 
+Vi vill att man som ingenjör ska kunna hitta sitt nästa IT jobb utan att behöva flytta eller ha onödigt långa ressträckor till och från sin arbetsplats. Det ska inte behöva vara en uppoffring att skaffa ett nytt jobb utan man ska enkelt och smidigt kunna välja något på ett behörigt avstånd inom IT branchen. 
+
+# Teknikstack
+-API: hanterar data på ett organiserat sätt
+-NodeJS, REST, JSON, JS, MongoDB (mLab)
+-App/Web application - hostas av IIS, Apache, Node
+-Hostas: Heroku
+-UI: JS (ECMA ES6), jQuery, CSS3, HTML5, Bootstrap
+-UX: Utseendet, varför, känslan
+-Deployment: Git, GitHub, Heroku
+
+## Grovplanering
+* Hämta jobb
+* Positionera appen
+* Utseende 
+* Skicka mail
+
+Planera: med GitHub Projects och Issues
+
+Visualisera: Genom att utföra tester och använda oss av wireframe eller dylikt
+
+Research: dokumentation av det vi finner görs här i GitHub i .md filer
+
+Kommunicera: Daily Scrum 09:00 - var ligger vi? vad ska ske under dagen?  
+
+Testa: ["Komma igång"](#Komma igång)
```

Comment:
This link doesn't seem to be working. I suggest you fix this before we merge it.